src/cli/eest/make/commands/test.py

```diff
@@ -65,47 +65,65 @@ def test():
     )
 
     fork_choices = [str(fork) for fork in get_forks()]
-    fork = input_select(
-        "Select the fork where this functionality was introduced", choices=fork_choices
+    fork = input_select("Select the fork", choices=fork_choices)
+
+    base_path = Path("tests") / fork.lower()
+    existing_dirs = [d.name for d in base_path.iterdir() if d.is_dir()]
+
+    location_choice = input_select(
+        "Select test directory",
+        choices=[
+            {"name": "Use current location", "value": "current"},
+            *existing_dirs,
+            {"name": "** Create new sub-directory **", "value": "new"},
+        ],
     )
 
-    eip_number = input_text("Enter the EIP number").strip()
-
-    # TODO: Perhaps get the EIP name from the number using an API?
-    eip_name = input_text("Enter the EIP name").strip()
-
-    test_name = eip_name.lower().replace(" ", "_")
-
-    file_name = f"test_{test_name}.py"
-
-    directory_path = Path("tests") / fork.lower() / f"eip{eip_number}_{test_name}"
-
-    file_path = directory_path / file_name
-
-    if file_path.exists():
+    if location_choice == "new":
+        eip_number = input_text("Enter the EIP number").strip()
+        eip_name = input_text("Enter the EIP name").strip()
+        directory_name = input_text("Enter directory name").strip()
+        module_name = input_text("Enter module name (snake_case)").strip()
+        dir_name = f"eip{eip_number}_{directory_name}"
```

Comment:
I think this is a bit misleading as-is. The user gets prompted for a `directory_name` but this doesn't reflect the name actually used.